src/Microsoft.AspNet.Server.Kestrel/Infrastructure/MemoryPoolIterator2.cs

```diff
@@ -537,7 +539,38 @@ public unsafe int Seek(ref Vector<byte> byte0Vector, ref Vector<byte> byte1Vecto
             }
         }
 
-        private static int FindFirstEqualByte(ref Vector<byte> byteEquals)
+        // Internal for testing
+        /// <summary>
+        /// Find first byte
+        /// </summary>
+        /// <param  name="byteEquals"></param >
+        /// <returns>The first index of the result vector</returns>
+        /// <exception cref="InvalidOperationException">byteEquals = 0</exception>
+        internal static int FindFirstEqualByte(ref Vector<byte> byteEquals)
```

Comment:
I have not problem with "for testing" comments (I was the one who suggested @Anderman should add it), but I meant it to be specifically for `FindFirstEqualByteSlow`. The point is that that method could/should be private, but we won't get any test coverage unless we call it explicitly unless we have any big endian test agents lying around.